docs/Tutorials/BeginnersTutorial.md

```diff
@@ -223,7 +223,9 @@ depending on what version of `Paraview` you have.)
 \image html beginners_paraview_top.png "Paraview top-bar"
 
```

Comment:
Commit message: You still have the fixup commit message in there. Could you remove that?

Reply:
Sure thing.